README.md

```diff
@@ -31,6 +31,11 @@ nuget {
     // nuget.exe version to use, defaults to 4.4.0
     // available versions can be found [here](https://dist.nuget.org/index.html)
     version = '4.4.0'
+
+    // Optionally you can set nuget location, which will be used for download:

```

Comment:
It may be good to say that this is mutually exclusive with the `version` parameter now, right?

Reply:
done